srcpkgs/gnupg2/template

```diff
@@ -17,6 +18,8 @@ license="GPL-3.0-or-later"
 homepage="https://www.gnupg.org/"
 distfiles="https://gnupg.org/ftp/gcrypt/gnupg/gnupg-${version}.tar.bz2"
 checksum=afa262868e39b651a2db4c071fba90415154243e83a830ca00516f9a807fd514
+build_options="ldap"
+build_options_default="${build_options}"
```

Comment:
This doesn't seem right as it will any build option that is ever added a default. Please change it to explicitely set the default build options.

Reply:
Fixed.